client/my-sites/domains/domain-management/domain-overview-pane/style.scss

```diff
@@ -211,7 +211,12 @@
 	.hosting-dashboard-item-view__content div {
 		flex-grow: 1;
 
+		.domain-forwarding-card__fields:last-child {
+			margin-bottom: 0;
+		}
+
 		.domain-forwarding-card__accordion .link-button {
+			margin-top: 1.5rem;
```

Comment:
I've noticed the margin is a bit too much when a notice is displayed.

<img width="564" alt="image" src="https://github.com/user-attachments/assets/76734995-99e3-47ba-8c5a-b1a20f479b3d" />

Reply:
Thanks! Updated it here - https://github.com/Automattic/wp-calypso/pull/99055/commits/84cc2f197ccb9efbfaa264c929a356088f301092. Merging it already to ship the PRs. But please give it a look post-merge